### PR TITLE
Update Fringe

### PR DESCRIPTION
**Pull request by SaraVieira**
- Updates fringe to latest version
- Removes icons from app and gets them from fringe
- Uses new bussiness button
- updates imports

### CODE REVIEW

File: components/ReportList.vue [hunk 1]
```diff
@@ -97,15 +97,12 @@ const getClasses = (dep) => {
         </div>
       </RadioGroup>
     </div>
-    <button
-      type="button"
+    <business-button
       :disabled="isLoading"
-      :class="[
-        'inline-flex items-center border border-transparent py-4 text-lg font-medium text-slate-50 disabled:opacity-60 disabled:cursor-default sticky bottom-0 w-full justify-center bg-green-600 hover:bg-green-700',
```
Comment (SaraVieira) [inline components/ReportList.vue:104]: Should we make a sucess version?

Comment (sarahghp) [reply components/ReportList.vue:104]: No, I think this should just be a one-off button. See the comments in https://github.com/axodotdev/turbo-axo/pull/161#issuecomment-1334102035 (& add your notes!)

For now, reverting this change should be ok.

File: components/ReportList.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@ import {
   RadioGroupDescription,
 } from "@headlessui/vue";
 import { useSingleReport, useCommit } from "../composables";
-import ShieldIcon from "./Icons/ShieldIcon.vue";
+import { BusinessButton, ShieldIcon } from "@axodotdev/fringe/lib";
```
Comment (sarahghp) [inline components/ReportList.vue:10]: I am so happy.

**Review by sarahghp — COMMENTED** [review]
Looks good to me, @SaraVieira. I think we can rever the change to the commit button and all is well.

**Review by sarahghp — APPROVED** [review]
Forgot to hit approve.

File: components/ReportList.vue (hunk 1 above)
Comment (SaraVieira) [reply components/ReportList.vue:104]: Reverted to just add the needed classes :)

File: components/ReportList.vue (hunk 2 above)
Comment (SaraVieira) [reply components/ReportList.vue:10]: I KNOWWWW, so clean!

**Pull request merged**